Makefile

```diff
@@ -147,9 +148,15 @@ tests-privileged:
 	$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C bpf cilium-map-migrate
 	$(MAKE) init-coverage
 	for pkg in $(patsubst %,github.com/cilium/cilium/%,$(PRIV_TEST_PKGS)); do \
-		PATH=$(PATH):$(ROOT_DIR)/bpf $(GO_TEST) $(TEST_LDFLAGS) $$pkg $(GOTEST_UNIT_BASE) $(GOTEST_COVER_OPTS) \
-		|| exit 1; \
-		tail -n +2 coverage.out >> coverage-all-tmp.out; \
+		if [ "${RACE}" == "" ]; then \
+			PATH=$(PATH):$(ROOT_DIR)/bpf go test -mod=vendor -tags lockdebug,privileged_tests $(TEST_LDFLAGS) $$pkg $(GOTEST_COVER_OPTS) \
```

Comment:
thus line should be kept has it was `PATH=$(PATH):$(ROOT_DIR)/bpf $(GO_TEST) $(TEST_LDFLAGS) $$pkg $(GOTEST_UNIT_BASE) $(GOTEST_COVER_OPTS)`

Reply:
Are you sure? the condition is `"${RACE}" == ""`

Reply:
I don't know about the `$(GOTEST_UNIT_BASE)` part, but I agree with @aanm that we want to keep `$(GO_TEST)`:

* The `privileged_tests` tag is always given in this target, see line 145 of of the patch.
* The `lockdebug` tag is enabled when `$(LOCKDEBUG)` is set, see Makefile.defs
* Makefile.defs also contains the logic for the `-mod=vendor` flag
* The `race` flag is enabled when  `$(RACE)` is defined, see Makefile.defs